### PR TITLE
Add column settings constants

### PR DESCRIPTION
**Pull request by ricco24**
For easy use in migration file.

```php
protected function up()
{
    $this->table('tableName')
         ->addColumn('title', Column::TYPE_STRING, [
             Column::SETTING_NULL => true, 
             Column::SETTING_DEFAULT => null
         ])
         ->addColumn('url', Column::TYPE_STRING, [
             Column::SETTING_AFTER => 'other_column',
             Column::SETTING_VALUES => ['one', 'two']
         ])
         ->addColumn('sorting', Column::TYPE_INTEGER)
         ->addColumn('created_at', Column::TYPE_DATETIME)
         ->create();
}
```

### CODE REVIEW

File: src/Database/Element/Column.php [hunk 1]
```diff
@@ -63,7 +87,7 @@ class Column
     /**
      * @param string $name name of column
      * @param string $type type of column
-     * @param array $settings - list of settings, available keys: null, default, length, decimals, signed, autoincrement, after, first
```
Comment (lulco) [inline src/Database/Element/Column.php:67]: I would like to keep it here O:-)

**Review by lulco — COMMENTED** [review]
Not sure if it is easier to write Column::SETTING_NULL => true :) but it is back compatible, so why not :)

File: src/Database/Element/Column.php (hunk 1 above)
Comment (ricco24) [reply src/Database/Element/Column.php:67]: But it is outdated ... and some setting options are missing. If this mr will be merged you can simply write Column::S [tab] and you see all available settings listed in your IDE.

Comment (lulco) [reply src/Database/Element/Column.php:67]: well, then we need to add missing options

Comment (lulco) [reply src/Database/Element/Column.php:67]: while I write "Column::S" I can write null twice ;)

Comment (ricco24) [reply src/Database/Element/Column.php:67]: If you know that some null setting exists ;-) I added comment back and add missing options.

**Review by lulco — COMMENTED** [review]
:+1:

**Pull request merged**